src/virtualenv/seed/embed/base_embed.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, options) -> None:
             warn(
                 "The --no-wheel option is deprecated. "
                 "It has no effect, wheel is no longer bundled in virtualenv. "
-                "This option will be removed in pip 26.",
+                "This option will be removed in virtualenv 26.",
```

Comment:
Do we ever need to remove it? There's no such thing as virtualenv 26. What are you trying to do here? Should we just remove this flag?

Reply:
Oh 🤦

Yeah, we don't need to ever remove it.